src/couch_views/src/couch_views_indexer.erl

```diff
@@ -78,10 +86,10 @@ init() ->
         fail_job(Job, Data, sig_changed, "Design document was modified")
     end,
 
-    Limiter = couch_rate:create_if_missing({DbName, DDocId}, "views"),
-
     State = #{
         tx_db => undefined,
+        tx_size => 0,
```

Comment:
Minor nit: would these make sense emitted as metrics. It could a separate PR though

Reply:
I was noodling over a way to dynamically enable the tracing logs I've been using while developing this. As metrics I think the cardinality would either be high enough it'd crush most metric collectors or coarse enough that it wouldn't be super helpful.

Reply:
Yeah, makes sense. I was thinking of low cardinality ones just see a range of transaction sizes and estimated limits, but you're right it would bounce around too much and not be very useful.